src-bin/ghci.hs

```diff
@@ -34,6 +37,10 @@ ghciArg = GhciArg
       help "The optional expression to evaluate once modules have successfully loaded" <>
       metavar "EXPR"
     ))
+  <*> optional (OP.switch
+    ( long "color" <>
+      help "The option to set colored output via -fdiagnostics-color=always"
+    ))
```

Comment:
Instead of `Maybe Bool` and then applying default in the code you can just apply the default here with `<> value True`.

Reply:
Why don't we take the color option as a string so we have the same options that ghci has?